contrib/platform/src/com/sun/jna/platform/win32/WinPerf.java

```diff
@@ -75,6 +84,106 @@
         public int UniqueID;
         public int NameOffset;
         public int NameLength;
+
+        public PERF_INSTANCE_DEFINITION() {
+            super();
+        }
+
+        public PERF_INSTANCE_DEFINITION(Pointer p) {
+            super(p);
+            read();
+        }
+    }
+
+    /**
+     * Describes object-specific performance information, for example, the
+     * number of instances of the object and the number of counters that the
+     * object defines.
+     * 
+     * @see <A HREF=
+     *      "https://msdn.microsoft.com/en-us/library/windows/desktop/aa373160(v=vs.85).aspx">
+     *      PERF_OBJECT_TYPE</A>
+     */
+    @FieldOrder({ "TotalByteLength", "DefinitionLength", "HeaderLength", "ObjectNameTitleIndex", "ObjectNameTitle",
+            "ObjectHelpTitleIndex", "ObjectHelpTitle", "DetailLevel", "NumCounters", "DefaultCounter", "NumInstances",
+            "CodePage", "PerfTime", "PerfFreq" })
+    public class PERF_OBJECT_TYPE extends Structure {
+        public int TotalByteLength;
+        public int DefinitionLength;
+        public int HeaderLength;
+        public int ObjectNameTitleIndex;
+        public int ObjectNameTitle; // always 32 bit
+        public int ObjectHelpTitleIndex;
+        public int ObjectHelpTitle; // always 32 bit
+        public int DetailLevel;
+        public int NumCounters;
+        public int DefaultCounter;
+        public int NumInstances;
+        public int CodePage;
+        public LARGE_INTEGER PerfTime;
+        public LARGE_INTEGER PerfFreq;
+
+        public PERF_OBJECT_TYPE() {
+            super();
+        }
+
+        public PERF_OBJECT_TYPE(Pointer p) {
+            super(p);
+            read();
+        }
+    }
+
+    /**
+     * Describes a performance counter.
+     * 
+     * @see <A HREF=
+     *      "https://msdn.microsoft.com/en-us/library/windows/desktop/aa373150(v=vs.85).aspx">
+     *      PERF_COUNTER_DEFINITION</A>
+     */
+    @FieldOrder({ "ByteLength", "CounterNameTitleIndex", "CounterNameTitle", "CounterHelpTitleIndex",
+            "CounterHelpTitle", "DefaultScale", "DetailLevel", "CounterType", "CounterSize", "CounterOffset" })
+    public class PERF_COUNTER_DEFINITION extends Structure {
+        public int ByteLength;
+        public int CounterNameTitleIndex;
+        public int CounterNameTitle; // always 32 bit
+        public int CounterHelpTitleIndex;
+        public int CounterHelpTitle; // always 32 bit
+        public NativeLong DefaultScale;
```

Comment:
In the header this is:
`LONG            DefaultScale; `
the other encounters of LONG are mapped to int (according to windows definition LONG is a 32 bit integer)

Reply:
OK, I've bookmarked [this page](https://docs.microsoft.com/en-us/windows/desktop/winprog/windows-data-types) so I'll stop guessing...